### ISSUE TITLE
cannot create a string longer than 0x3fffffe7 characters

### ISSUE THREAD

**Issue by bwgjoseph** [whitecolor/ts-node-dev#238]
Hi,

This problem kinda came out of outwhere, I was using as usual, and after some commits, and when I try to start my app, it throws me this error (cannot create a string longer than 0x3fffffe7 characters)

Now, even if I checkout to older commits, the error persist

I'm using `ts-node-dev 1.0.0-pre.56`, when I upgrade to `1.1.1`, it seem to be working fine though. However, I'm trying not to upgrade because of the console color output issue described in #207. 

I switched to `ts-node` and it is able to run. I tried with `--debug` options and it just states

```js
[DEBUG] Child error
[ERROR] Error: Cannot create a string longer than 0x3fffffe7 characters
[DEBUG] Disconnecting from child
[DEBUG] Sending SIGTERM kill to child pid xxx
[DEBUG] Child exited with code null
```

Anyway to resolve this without upgrading to the latest version?

Thanks

**Comment by whitecolor** [comment]
If you could create a repro example of the problem with colors?

**Comment by bwgjoseph** [comment]
Hi @whitecolor,

Thanks, I should have done that earlier. I created a repro, and somehow figured out the cause and I traced it to [this commit](https://github.com/whitecolor/ts-node-dev/commit/5540ac38a526c588a43a63db6a46af35ff13de39#diff-c60d44e47c8429d1d31fd42db19599ed941fe6edc51ea333d97dd639a2e40d49).

As my logger has some configuration that depends on the `NODE_ENV` and when that commit took away `development` environment, the value of `process.env.NODE_ENV` is undefined, and hence, set to a different output which does not colorize the console output. I played around with it, and by manually setting the `NODE_ENV` to `development`, the log output as expected.

On that note, any reason why was it being removed?

---

Also, for the original problem stated in the thread, the issue also was resolved by using a different directory. Somehow, the directory was "corrupted"? Assuming this "corrupted" directory is called `server`

1. I moved this `server` directory to some other location, and tried to run `npm ci` and start the server, it throws me the error
2. So I cloned a new copy from my repo, and named it `server`, and install my packages, and start the server without issue

I'm not exactly sure what causes the "corruption"

Thanks
